app/src/main/java/edu/usf/sas/pal/muser/util/PreferenceUtils.java

```diff
@@ -0,0 +1,24 @@
+package edu.usf.sas.pal.muser.util;
```

Comment:
Doesn't seem like this class is used currently?

Reply:
Correct - It's not being used yet. I have written the functions to save Strings to the `SharedPreferences` and never actually called them anywhere.